src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -1,52 +1,78 @@
-import { Box, Button, Divider, Grid, TextField, useMediaQuery, useTheme } from '@mui/material';
+import { Alert, Box, Divider, Grid, Snackbar, TextField, useMediaQuery, useTheme } from '@mui/material';
 import { Field, Formik } from 'formik';
+import { LoadingButton } from '@mui/lab';
 import { PasswordInput } from '../PasswordInput/PasswordInput';
-import { RegisterFormErrors } from './RegisterForm.types';
+import { RegisterFormType } from './RegisterForm.types';
 import styles from './RegisterForm.module.scss';
+import { SyntheticEvent } from 'react';
+import { useCreateUserMutation } from '../../common/services/registerUserRequest';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
 export function RegisterForm() {
+  const [open, setOpen] = useState(true);
```

Comment:
I think we do not need this and we can base on mutation-hook state

Reply:
This is necessary for closing snackbars. If we do not have this, user cannot close a snackbar manually, and will have to wait his allotted time to be redirected. 

I'll remove this, if closing snackbars is not a necessary feature.